src/main/java/com/codeborne/selenide/conditions/IsImage.java

```diff
@@ -0,0 +1,22 @@
+package com.codeborne.selenide.conditions;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Driver;
+import org.openqa.selenium.WebElement;
+
+public class IsImage extends Condition {
+
+  public IsImage() {
+    super("is image");
+  }
+
+  @Override
+  public boolean apply(Driver driver, WebElement webElement) {
+    if (!"img".equalsIgnoreCase(webElement.getTagName())) {
+      return false;
+    }
+    return driver.executeJavaScript("return arguments[0].complete && " +
+      "typeof arguments[0].naturalWidth != 'undefined' && " +
+      "arguments[0].naturalWidth > 0", webElement);
+  }
+}
```

Comment:
Now we have code duplication in `com.codeborne.selenide.commands.IsImage` and `com.codeborne.selenide. conditions.IsImage`. I think one of them should use the other.

Reply:
Indeed! 
Will it be ok if I rename `conditions.IsImage` to `IsImageLoaded` and inside create public static method `isImage()` and use this method from commands.IsImage? 

Renaming will help to not write the full classpath (with package name)

So in commands.IsImage it will look like this: `return IsImageLoaded.isImage(locator.driver(), img);` ?

Reply:
Yes, why not.

Reply:
done